File: plugin_fly_me/src/main/java/com/fly/tour/me/MeApplication.java
package com.fly.tour.me;

import com.fly.tour.common.BaseApplication;
import com.fly.tour.common.manager.NewsDBManager;

/**
 * Description: <MeApplication><br>
 * Author:      mxdl<br>
 * Date:        2020/3/4<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
public class MeApplication extends BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        NewsDBManager.getInstance(this).initNewsDB();
    }
}

File: plugin_fly_me/src/main/java/com/fly/tour/me/MeApplication.kt
package com.fly.tour.me

import com.fly.tour.common.BaseApplication
import com.fly.tour.common.manager.NewsDBManager

/**
 * Description: <MeApplication><br></br>
 * Author:      mxdl<br></br>
 * Date:        2020/3/4<br></br>
 * Version:     V1.0.0<br></br>
 * Update:     <br></br>
</MeApplication> */
class MeApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        NewsDBManager.getInstance(this)!!.initNewsDB()
    }
}